### DISCUSSION TITLE
Google Colab Session Crash

### DISCUSSION THREAD

**Discussion by MQasimJalil** [myshell-ai/OpenVoice#277] [Q&A]
I am trying to get this to work on Google Colab but the notebook crashes every time saying the session crashed for an unknown reason.

This happens when I run: **"target_se, audio_name = se_extractor.get_se(reference_speaker, tone_color_converter, vad=False)"** 

If anyone knows the solution to this let me know any help you can give is appreciated.

**Comment by MatanYemini** [comment]
I have the same issue.

**Reply by MatanYemini** [reply]
@MQasimJalil Seems like CUDA is not installed correctly in colab / sagemaker by default. I have changed line 22 in `se_extractor.py` from "cuda" to "cpu" and from float16 to float32...

**Reply by krylor** [reply]
Did that work for you?

**Reply by MQasimJalil** [reply]
I wanted something faster so I went ahead with other models available

**Reply by MatanYemini** [reply]
@MQasimJalil - what models are you using instead? :) 
@krylor - after changing - it worked

**Reply by MQasimJalil** [reply]
Coqui.ai

**Reply by MatanYemini** [reply]
@MQasimJalil but they claim to shutdown 🤔 . Are you still recommending it?

**Reply by MQasimJalil** [reply]
Nah, I am just experimenting and it works now. That's good enough for me please look around for better options if you are working on some serious project.
